optuna/storages/_rdb/storage.py

```diff
@@ -145,6 +145,21 @@ def objective(trial):
         https://docs.sqlalchemy.org/en/13/core/engines.html#sqlalchemy.create_engine.params.
         pool_pre_ping
 
+    .. note::
+        Mainly in a cluster environment, running trials are often killed unexpectedly by the job
+        scheduler. If you want to monitor the trial's life and death, please use the heartbeat
+        mechanism. Set ``heartbeat_interval``, ``grace_period``, and `failed_trial_callback``
+        appropriately according to your use case. For more details, please refer to the
+        `FAQ page`_ and `Example page
+        <https://github.com/optuna/optuna-examples/blob/main/pytorch/pytorch_checkpoint.py>`_.
+
+    .. _FAQ page:
+        https://optuna.readthedocs.io/en/latest/faq.html#can-i-monitor-trials-and-make-them-
+        failed-automatically-when-they-are-killed-unexpectedly
+
+    .. seealso::
+        See :class:`~optuna.storages.RetryFailedTrialCallback` for heartbeat mechanism.
```

Comment:
This doesn't make sense to me a bit. I think it'd be better to contain information that `we can retry failed trials` (e.g. `You can use RetryFailedTrialCallback to automatically retry failed trials detected by heartbeat`).